calico-enterprise/network-policy/policy-tiers/tiered-policy.mdx

```diff
@@ -133,36 +141,20 @@ The following diagrams show the relationship between all of the elements that af
 
 ### Implicit default deny
 
-As shown in the following diagram, at the end of each tier is an implicit default deny. This is a safeguard that helps mitigate against unsecured policy. Because of this safeguard, you must explicitly apply the **Pass** action rule when you want traffic evaluation to continue. In the following example, the Pass action in a policy ensures that traffic evaluation continues, and overrides the implicit default deny.
+As shown in the following diagram, at the end of each tier is an implicit default deny. This is a safeguard that helps mitigate against unsecured policy.
+To override the implicit default deny, and to allow traffic evaluation to continue, you can change tier's default action to **Pass**.
 
 ![implicit-deny](/img/calico-enterprise/implicit-deny.svg)
 
-Let’s look at a Dev/Ops global network policy in a high precedence tier (Platform). The policy denies ingress and egress traffic to workloads that match selector, `env == "stage"`. To ensure that policies continue to evaluate traffic after this policy, the policy adds a Pass action for both ingress and egress.
-
-**Pass action rule example**
 
 ```yaml
 apiVersion: projectcalico.org/v3
-kind: GlobalNetworkPolicy
+kind: Tier
 metadata:
-  name: devops.stage-env
+  name: security
 spec:
-  tier: devops
-  order: 255
-  selector: env == "stage"
-  ingress:
-    - action: Deny
-      source:
-        selector: env != "stage"
-    - action: Pass
-  egress:
-    - action: Deny
-      destination:
-        selector: env != "stage"
-    - action: Pass
-  types:
-    - Ingress
-    - Egress
+  order: 300
+  defaultAction: Pass
```

Comment:
No, with the `defaultAction` added, there is no need to add rules with Pass action just to prevent hitting the implicit deny. I think the previous method looked like a hack, and this version is cleaner.

Reply:
I understand that we have a better approach, but I mean we have two way now. At the least, We should add that the previous method still works, but is not recommended. There are people who will upgrade and will not change their rules and may get confused if something does not work, that they are doing it wrong. WDYT?

Reply:
It makes sense to document both options. Adding a pass rule is more specific than having default pass action.